composition/src/errors/errors.ts

```diff
@@ -709,7 +709,7 @@ export function undefinedFieldInFieldSetErrorMessage(
 ): string {
   return (
     ` The following field set is invalid:\n  "${fieldSet}"\n` +
```

Comment:
Here lowercased

Reply:
Never mind—looks like I chose to keep field set lower case.